tests/testthat/test-project-elements.R

```diff
@@ -0,0 +1,186 @@
+parent_element <- list()
+# Tests for add project element 
+test_that('project function errors when missing mandatory identifier inputs', {
+  expect_error(add_project(parent_element = parent_element,
+                           award_information = list(),
+                           project_personnel = list()),
+               "Please provide the project title")
+  
+  expect_error(add_project(parent_element = parent_element, 
+                           project_title = "my project title", 
+                           project_personnel = list()),
+               "Please provide the award information.")
+  
+  expect_error(add_project(parent_element = parent_element,
+                           project_title = "my project title", 
+                           award_information = list()), 
+               "Please provide the project personnel")
+})
+
+test_that('arguments have all the nessisary required values', {
+  expect_error(add_project(parent_element = parent_element,
```

Comment:
After you add the default funder IDs (BOR, CDFW, CDWR), please update a few of the test to use them. unit tests are the most reliable documentation, it helps to make the "fake" arguments as realistic as possible.